projects/ADuCM3029_demo_cn0428_cn0429/src/adi_m355_gas_sensor.cpp

```diff
@@ -232,18 +214,16 @@ SENSOR_RESULT M355_GAS::SetI2CAddr(uint8_t new_I2C_address)
 	 */
 	/* Send command to change I2C address */
 	Result = this->open(); /* uses default address */
-	if (Result != SENSOR_ERROR_NONE)
-		return Result;
+
 	delay_ms(50);
 	Result = this->I2CReadWrite(WRITE, SET_I2C_ADDRESS, pBuff, 1);
-	if (Result != SENSOR_ERROR_NONE)
-		return Result;
-
-	this->close();
+	/* Do not raise error in case of soft reset (address set previously) */
```

Comment:
If `Result` is not tested here anymore do we still need it?

Reply:
Result is still tested later in the same function. Can we keep it?

Reply:
Yes, we can keep it. I would say it is up to you. There are some instances it which errors might be masked because the `Result` is attributed but not tested, but if you are pretty sure they will not hinder the debugging process in the future then it's fine to leave it like it is.